projects/packages/my-jetpack/_inc/data/products/use-products-that-require-user-connection.ts

```diff
@@ -0,0 +1,26 @@
+import { PRODUCT_STATUSES } from '../../components/product-card';
+import { useAllProducts } from './use-product';
+import type { StateProducts } from '../types';
+
+const useProductsThatRequireUserConnection: () => Array< keyof StateProducts > = () => {
```

Comment:
it's more of `useProductSlugsThatRequireUserConnection` 🤔 

also, not sure if that should be a hook instead of helper function, as we don't do anything special here rather than parsing data. what do you think?

Reply:
Good point, I was actually thinking this when moving it over but forgot